src/common/input/DInputKeyboardMouse.cpp

```diff
@@ -0,0 +1,262 @@
+// This is an open source non-commercial project. Dear PVS-Studio, please check it.
+// PVS-Studio Static Code Analyzer for C, C++ and C#: http://www.viva64.com
+// ******************************************************************
+// *
+// *  This file is part of the Cxbx project.
+// *
+// *  Cxbx and Cxbe are free software; you can redistribute them
+// *  and/or modify them under the terms of the GNU General Public
+// *  License as published by the Free Software Foundation; either
+// *  version 2 of the license, or (at your option) any later version.
+// *
+// *  This program is distributed in the hope that it will be useful,
+// *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+// *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// *  GNU General Public License for more details.
+// *
+// *  You should have recieved a copy of the GNU General Public License
+// *  along with this program; see the file COPYING.
+// *  If not, write to the Free Software Foundation, Inc.,
+// *  59 Temple Place - Suite 330, Bostom, MA 02111-1307, USA.
+// *
+// *  (c) 2019 ergo720
+// *
+// *  All rights reserved
+// *
+// ******************************************************************
+
+// Copyright 2010 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
```

Comment:
If you don't mind my asking... what `license.txt` file? The license file we have at the moment is `COPYING`. Since we're still using the same license perhaps change the sentence to
```suggestion
// Refer to top of this file's copyright license.
```

or to the `COPYING` file.

Though, I think we should rename `COPYING` to `LICENSE`.

Reply:
That file is of Dolphin, not Cxbx-R. I just copied their copyright header.

Reply:
If you want to refer to their own file, then you should use a direct link. Since `license.txt` file isn't included in Cxbx-R nor is in the pull request.

Does it make sense?